3scale/porta
# [assets] Fixing versions of Bootstrap 3.0.0 in dev portal

**What this PR does / why we need it**:

Fixes a bug in the bootstrap libs we are providing in dev portal. 

**Which issue(s) this PR fixes** 
Fixes https://issues.jboss.org/browse/THREESCALE-1067 
NOTE: A migration of those files will still be needed for SaaS.

**Verification steps** 

0. Make sure you replace the assets stored in ddbb with the new ones. Easy way to do it: Go to Decveloper Portal -> Content -> bootstrap.min.js replace with the new one.
1. Visit Dev Portal in a mobile device or browser with mobile dimensions.
2. Play with the burger button, it should work as expected.

**Special notes for your reviewer**:
* An upgrade of the files in this PR should be schedule for SaaS.
* Not possible to upgrade to a higher version since will require jquery upgrade, and will break things.
* Ugly fix, as our dev portal itself.

** Upgrade partials in SaaS or a running onprem **:
```
bundle exec rake 'cms:update_builtin_partials[0, "javascripts/bootstrap.js"]'
bundle exec rake 'cms:update_builtin_partials[0, "javascripts/bootstrap.min.js"]'
```
